addons/danger/functions/fnc_indoor.sqf

```diff
@@ -15,4 +15,6 @@
  * Public: No
 */
 params ["_unit"];
-lineIntersects [eyePos _unit, eyePos _unit vectorAdd [0, 0, 4]]
+private _trace = lineIntersectsSurfaces [eyePos _unit, eyePos _unit vectorAdd [0, 0, 4], _unit, objNull, true, 1, "GEOM", "NONE"];
```

Comment:
4 Meters up will work on most residential buildings, but it will not work on warehouses and the like with higher ceiling

Reply:
i would say 6 meters.

Reply:
Will fix

Reply:
i would also change it so that we return all hits and filter them for houses with findIf. so that if you forexample are laying under a car inside a garage it still sees that you are in a building.

Reply:
so `_trace findIf {_x select 3 isKindOf "Building"};`